R/skim_print.R

```diff
@@ -61,6 +61,27 @@ print.skim_df <- function(x,
   }
 }
 
+#' @export
+print.one_skim_df <- function(x,
```

Comment:
We don't have a summary method for a one_skim_df. Can we remove the logic for the summary and instead have this?

```
print.one_skim_df <- function(x, n, width, ...) {
  NextMethod("print")
}
```

Reply:
I took the logic of 
https://github.com/ropensci/skimr/blob/562b75342fa31f186106a6aa17c5aef3df897f97/R/skim_print.R#L38
but took out the purrr.

I'm not sure but now that we are not silently getting the warning it may be that other parts are getting hit in new ways.

Reply:
Maybe just make `include_summary = FALSE `?